src/Pusher.php

```diff
@@ -55,6 +55,9 @@ class Pusher implements LoggerAwareInterface
      *                         useTLS - quick option to use scheme of https and port 443.
      *                         encrypted - deprecated; renamed to `useTLS`.
      *                         cluster - cluster name to connect to.
+     *                         encryption_master_key - the encryption master key used to encrypt events' data for private-encrypted channels
+     *                         debug - debug mode
```

Comment:
What does "debug mode" mean? I'm guessing it's verbose logging, in which case we should be explicit about that in the documentation of this option.

Reply:
Yes, it makes sense to be more explicit with that. Thanks 👍